### DISCUSSION TITLE
Filament 3: Nofications don't appear

### DISCUSSION THREAD

**Discussion by monointeractive** [filamentphp/filament#8577] [General]
Hello. I'm trying to use the notifications package using only Filament Table Builder v3 and Filemanet Notifications v3 (without Filament Panel). Unfortunately they don't appear. Can anyone help me? :)

```
[..]->action(function(){  
    Notification::make()
        ->title('Hello world')
        ->success()
        ->send();
}),
```

**Comment by leandrocfe** [comment]
Did you do this [setup](https://filamentphp.com/docs/3.x/notifications/installation)?
Did you add `@livewire('notifications')` in your Blade layout?

**Reply by monointeractive** [reply]
Yes, I tried that and it doesn't work for me - but I still can't find this directive in the documentation.

**Reply by leandrocfe** [reply]
[here](https://filamentphp.com/docs/3.x/notifications/sending-notifications)
![image](https://github.com/filamentphp/filament/assets/3833889/de0b11c8-91bc-4fc9-9602-d8f6f5930ca0)

**Reply by leandrocfe** [reply]
This was supposed to work

**Reply by monointeractive** [reply]
Thanks, I know that's why it didn't work. After the notification, there was a reload. The question is whether I can see this notification after executing $this->redirect(..?

**Reply by leandrocfe** [reply]
why not a [livewire event](https://livewire.laravel.com/docs/events) to refresh the component?

**Reply by monointeractive** [reply]
I don't understand how these events could be used in the context of displaying after redirect notifications.

But since notifications are stored in the session (flash), is there no way to display them after performing the reload action?
